docs/src/main/asciidoc/_elytron/OpenSSL.adoc

```diff
@@ -49,4 +51,50 @@ compared to TLSv1.2. For this reason, the use of TLSv1.3 is currently disabled b
 by configuring the `cipher-suite-names` attribute for an SSL context. See the sections on
 <<configuring-a-server-sslcontext,Configuring a server SSLContext>> and
 <<configuring-a-client-sslcontext,Configuring a client SSLContext>> for more details. It is recommended to test
-for performance degradation prior to enabling TLSv1.3 in a production environment.
\ No newline at end of file
+for performance degradation prior to enabling TLSv1.3 in a production environment.
+
+== Adding Additional Native Libraries
+
+By default, WildFly includes WildFly OpenSSL native libraries for commonly used platforms (Linux x86_64, Windows x86_64,
+Mac OS X x86_64, and Linux s390). However, the https://github.com/wildfly-security/wildfly-openssl-natives[WildFly OpenSSL Natives project]
+contains modules that can be used to build WildFly OpenSSL native libraries for other platforms as well. The steps needed to build and make
+use of a WildFly OpenSSL native library for a different platform than the ones provided by default will be described below.
+
+=== Building the Native Library
+
+Make sure you have cloned the https://github.com/wildfly-security/wildfly-openssl-natives[WildFly OpenSSL Natives] project
+locally. We'll use `$WILDFLY_OPENSSL_NATIVES` to denote the path to this project. Next, `cd` to the module that you want
+to build. In this example, we'll use the `linux-i386` module but similar steps can be used to build and make use of another
+module instead.
+
+[source,shell]
+----
+cd $WILDFLY_OPENSSL_NATIVES/linux-i386
+mvn clean install
+----
+
+Notice that this now results in a `wildfly-openssl-linux-i386-VERSION.jar` in the `$WILDFLY_OPENSSL_NATIVES/linux-i386/target` directory.
+We'll make use of this in the next step.
+
+=== Adding a New Module to WildFly
+
+First, add a new module that contains the WildFly OpenSSL Java artifact:
```

Comment:
Should this say anything about how this is basically overriding the existing module? That's a subtlety that may be useful to know if something goes wrong, or even just to help people understand better what's going on.

Reply:
Good point, I've updated this section to mention that. Thanks!